skynet/http_client.py

```diff
@@ -33,4 +33,17 @@ async def post(url, **kwargs):
         return await response.json()
 
 
-__all__ = ['get']
+async def request(method, url, **kwargs) -> aiohttp.ClientResponse:
+    session = _get_session()
+
+    return await session.request(method, url, **kwargs)
+
+
+async def close():
+    session = _get_session()
+
+    if session is not None:
+        await _session.close()
```

Comment:
shouldn't we set it to None after closing?

Reply:
i guess it doesn't really matter since this happens before the app closes

Reply:
Right, I noticed that after seeing where it's used. Still, it IMHO makes sense to have its functionality be explicitly correct, not implicitly because of how we use it.